src/openforms/prefill/base.py

```diff
@@ -89,3 +89,7 @@ def get_identifier_value(
             and submission.auth_info.attribute == cls.requires_auth
         ):
             return submission.auth_info.value
+
+    @property
+    def extra_data(self):
+        return None
```

Comment:
```suggestion
    @property
    def extra_data(self) -> JSONObject | None:
        return None
```
to indicate that it will be some kind of top-level JSON object.

Probably this could also be a `classmethod` rather than `property` since there's nothing of value inside an instance usually (see https://stackoverflow.com/a/57055258)